views/dashboard/signatories.ejs

```diff
@@ -7,7 +7,7 @@
     <div class="signatory-panel">
         <% signatories.forEach(sig => { %>
             <p>
-                <a href="https://stackexchange.com/users/<%= sig.se_acct_id %>?tab=accounts">
+                <a href="https://stackexchange.com/users/<%= sig.se_acct_id %>?tab=accounts" target="_blank">

```

Comment:
Good thought. Also needs `rel="noopener noreferrer"` added to avoid reverse tabnabbing, though.

Reply:
Updating with these attributes now and reading about this reverse tabnabbing.